include/mbedtls/ssl.h

```diff
@@ -1593,6 +1610,14 @@ struct mbedtls_ssl_context
 #endif
 };
 
+#define MBEDTLS_SSL_PEND_FATAL_ALERT( type, user_return_value )         \
```

Comment:
Agree.

Reply:
Done. And move the macro to `ssl_misc.h` . I think it should be used internally now.

Reply:
I am not convinced this saves something (you have to prepare arguments and invoke the function), and it's not part of the 1.3 guidelines. Please, can we _not_ spend time with such subjective things for now...

Reply:
I seem to remember that at some point (in baremetal or in the code size mini-hackaton we had in Cambridge), you (Hanno) tried both and noticed the macro actually led to a slightly smaller code size. I might be misremembering though.

Anyway, I agree now is not the best time to think about that. Not only because I agree there are other more important things to focus on, but also because the question will be much easier to answer when the macro/function is actually used all over the place: then we can do measurements and see what the results are. In the meantime, I suggest keeping whatever version's currently in the PR and not spending more time on this.

Reply:
I don't remember actually measuring, but intuitively it's seems likely: We have 3 stores vs. preparing arguments (which requires freeing up registers) and branching. 

I think it's worth investigating in some detail, but as you say @mpg, now is not the time.